### PR TITLE
Element isMany only if maxOccurs > 1 or Unbounded

### PR DESCRIPTION
**Pull request by Glavin001**
Reapplying fix from https://github.com/strongloop/strong-soap/commit/9834873440cd7f239b063c33ebea0f7525f85d85

Broken by https://github.com/strongloop/strong-soap/commit/d37b59ad7f1198c8942936365ae37932db42acf7

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by Glavin001** [comment]
Please let me know if there's anything else you need. I have signed the Contributor License Agreement already. My team needs this bug fixed so we can use Strong-SOAP. It looks like an obvious error so I do not expect this will take too much of your time to review 😜 . Thank you in advance!

**Comment by Glavin001** [comment]
Added another fix. Looks like there were a couple bugs created from https://github.com/strongloop/strong-soap/commit/d37b59ad7f1198c8942936365ae37932db42acf7 :-1:. Catching them one by one!

File: src/parser/xsd/element.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Element extends XSDElement {
     var descriptor = this.descriptor =
       new XSDElement.ElementDescriptor(qname, typeQName, form, isMany);
 
-    if (this.$nillable) {
+    if (this.$nillable === true || this.$nillable === "true") {
```
Comment (Glavin001) [inline src/parser/xsd/element.js]: `this.$nillable` was `false` which is truthy, as a string. Made the conditional more explicit so there are no false positives.

File: src/parser/xsd/element.js [hunk 2]
```diff
@@ -41,9 +41,6 @@ class Element extends XSDElement {
       var refDescriptor = this.ref.describe(definitions);
       if (refDescriptor) {
         this.descriptor = descriptor = refDescriptor.clone(isMany);
-        if (this.$nillable) {
```
Comment (Glavin001) [inline src/parser/xsd/element.js]: This is redundant. It would already be run by the same code above, which is not nested inside of another conditional.

**Comment by TheDeathStreaker** [comment]
Can someone please merge this PR since this is really a critical bug and our team using strong-soap library also needs it fixed.

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by raymondfeng** [comment]
@Glavin001 @nyteriven Sorry for the delay. 

@nyteriven Can you sign the [CLA](https://cla.strongloop.com/agreements/strongloop/strong-soap)?

**Comment by TheDeathStreaker** [comment]
@Glavin001 could you somehow please get @nyteriven to sign the [https://cla.strongloop.com/agreements/strongloop/strong-soap](CLA). Since our team really needs that fixed and we can't just manually fix it because we have automated builds for project

**Comment by kjdelisle** [comment]
@nyteriven @Glavin001 While we're waiting for the CLA to be signed, would you be able to fix the commit messages to conform to our guidelines?

See [our commit guide] (https://loopback.io/doc/en/contrib/git-commit-messages.html)

File: src/parser/xsd/element.js (hunk 1 above)
Comment (sieteunoseis) [reply src/parser/xsd/element.js:42]: can we get this fixed? it solved my issue

**Pull request closed without merging**